src/components/external_fields/external_fields.stories.mdx

```diff
@@ -64,26 +61,24 @@ List of key-value fields with formatting support.
 ```
 
 <Canvas>
-  <Story name="Default">
-      {Default.bind({})}
-  </Story>
+  <Story name="Default">{Default.bind({})}</Story>
 </Canvas>
 
 ## API
 
 ### Attributes/Properties
 
-| property | type | default |
-| -------- | ---- | ------- |
-| fields[].key | string | |
-| fields[].value | number, string | |
-| format [key] | string | |
-| format [value] | "centsToDollars", "M/d/yyyy" | |
+| property       | type                         | default |
```

Comment:
Note for later that we should find tools to help us write md tables easier; even better if we can dynamically generate them somehow.

Reply:
oh I wonder what happened here... maybe my editor didnt recognize the file type at first... the others I edited formatted themselves 😬

Reply:
My editor's gotta eat what your editor's having!